Makefile

```diff
@@ -12,7 +12,46 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+.PHONY: push-multiarch-% push-multiarch
+
 CMDS=csi-node-driver-registrar
 all: build
 
 include release-tools/build.make
+
+# This target builds multiarch images using Moby BuildKit builder toolkit.
+# Docker Buildx is included in Docker 19.03 and needs DOCKER_CLI_EXPERIMENTAL enabled to run corresponding commands.
+# Currently amd, s390x and Windows manifest is pushed for canary, release branch and released tags.
+# Images generated from Prow build are pushed to staging area on gcr
+push-multiarch-%:
+	make BUILD_PLATFORMS="windows amd64 .exe"
+	set -ex; \
+	gcloud auth configure-docker; \
+	DOCKER_CLI_EXPERIMENTAL=enabled; \
+	export DOCKER_CLI_EXPERIMENTAL; \
+	docker buildx create --use --name multiarchimage-buildertest; \
+	pushMultiArch () { \
+                tag=$$1; \
+                docker buildx build --push -t $(IMAGE_NAME):amd64-linux-$$tag --platform=linux/amd64 -f $(shell if [ -e ./cmd/$*/Dockerfile.multiarch ]; then echo ./cmd/$*/Dockerfile.multiarch; else echo Dockerfile.multiarch; fi) --label revision=$(REV) .; \
```

Comment:
Perhaps create a local variable `dockerfile=$(shell if [ -e ./cmd/$*/Dockerfile.multiarch ]; then echo ./cmd/$*/Dockerfile.multiarch; else echo Dockerfile.multiarch; fi)` above and use it here and below?

Reply:
I think we can use this once windows is also a part of `Dockerfile.multiarch`. otherwise we have to set an extra variable for windows dockerfile too.

Reply:
Any way we can make the list of linux platforms as a variable instead of hard coding the command for each platform?

Unfortunately I don't think there's much we can do to generalize windows atm.

Reply:
Currently with the help of this PR we are just testing that whether push-multiarch (keeping cloudbuild minimal) can be used to correctly tag multiarch images.
Once all these changes are working fine as expected then we can modify docker buildx command as `docker buildx build --push -t $(IMAGE_NAME):$$tag --platform=linux/amd64,linux/s390x -f ...` option which will have single docker buildx command for linux (with multiaple arch)